DQMOffline/L1Trigger/python/L1TEfficiencyHarvesting_cfi.py

```diff
@@ -1,14 +1,22 @@
 import FWCore.ParameterSet.Config as cms
 
 l1tEfficiencyHarvesting = cms.EDAnalyzer("L1TEfficiency_Harvesting",
-
-  verbose  = cms.untracked.bool(False),
-  plotCfgs = cms.untracked.VPSet(
-    cms.untracked.PSet( dqmBaseDir = cms.untracked.string("L1T/Efficiency/Muons"),
-                        plots      = cms.untracked.vstring("EffvsPt16","EffvsEta16","EffvsPhi16",\
-                                                           "EffvsPt20","EffvsEta20","EffvsPhi20",\
-                                                           "EffvsPt25","EffvsEta25","EffvsPhi25")
-                      )
-  )
-                                         
+    verbose  = cms.untracked.bool(False),
+    plotCfgs = cms.untracked.VPSet(
+        cms.untracked.PSet( dqmBaseDir = cms.untracked.string("L1T/Efficiency/Muons"),
```

Comment:
DO NOT use tabs inside python files. Use 4 spaces.
Also, https://www.python.org/dev/peps/pep-0008/